src/handler/policies/trustChainsHandler.ts

```diff
@@ -3,14 +3,29 @@ import Response from 'src/response'
 import { NextApiResponse } from 'next'
 import { Tag } from 'src/config/guardianTags'
 import trustChainMapper from 'src/mappers/trustChainMapper'
+import { AccountLoginResponse } from 'src/guardian/account'
+import config from 'src/config'
 
 async function TrustChainsHandler(
 	req: GuardianMiddlewareRequest,
 	res: NextApiResponse
 ) {
 	const { policyId } = req.query
 	const { guardian } = req.context
-	const { accessToken } = req
+
+	let { accessToken } = req
```

Comment:
In the house of DOVU we don't use lets, transform this into a separate method or inline function

Reply:
Let us not see let again. It has been squashed.